### ISSUE TITLE
Song does not stop playing after closing spotify-qt

### ISSUE THREAD

**Issue by marco-calautti** [kraxarn/spotify-qt#274]
### System information
Operating System: Arch Linux 
KDE Plasma Version: 6.2.4
KDE Frameworks Version: 6.8.0
Qt Version: 6.8.0
Kernel Version: 6.12.1-arch1-1 (64-bit)
Graphics Platform: Wayland
Processors: 12 × Intel® Core™ i7-10750H CPU @ 2.60GHz
Memory: 31.1 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics
Manufacturer: Dell Inc.
Product Name: XPS 15 9500

### Detailed information
When playing a song over librespot (dunno about spotifyd), if I close spotify-qt the song keeps playing in the background.

**Comment by kraxarn** [comment]
Did you start librespot from spotify-qt itself? If you start it yourself outside of the application, it's not supposed to pause the playback when it's closed, as neither does the official client.

**Comment by marco-calautti** [comment]
Yes, I started librespot from outside spotify-qt, because spotify-qt is not able to use the library by itself.

**Comment by kraxarn** [comment]
You can set it up from within spotify-qt if you go to "Settings > Spotify" and enter the path for the librespot binary. That way, it closes together with spotify-qt, and stops the music playback. Running it externally makes it difficult for spotify-qt to know if it should stop the music or not when closed.

Closing as this is the intended behaviour.
